content/sensu-core/0.29/quick-start/five-minute-install.md

```diff
@@ -88,50 +80,70 @@ protected-mode no{{< /highlight >}}
 sudo systemctl enable redis
 sudo systemctl start redis{{< /highlight >}}
 
-**5. Install Sensu:**
+**5. Install and start RabbitMQ:**
 
-For Sensu Core:
+Add the RabbitMQ Erlang repository (required for RabbitMQ):
 
 {{< highlight shell >}}
-sudo yum install sensu uchiwa -y{{< /highlight >}}
+echo ' [rabbitmq-erlang]
+name=rabbitmq-erlang
+baseurl=https://dl.bintray.com/rabbitmq/rpm/erlang/20/el/7
+gpgcheck=1
+gpgkey=https://www.rabbitmq.com/rabbitmq-release-signing-key.asc
+repo_gpgcheck=0
+enabled=1' | sudo tee /etc/yum.repos.d/rabbitmq-erlang.repo{{< /highlight >}}
 
-Or for [Sensu Enterprise][9]:
+Install Erlang (required for RabbitMQ):
 
 {{< highlight shell >}}
-sudo yum install sensu sensu-enterprise sensu-enterprise-dashboard -y{{< /highlight >}}
+sudo yum install erlang -y{{< /highlight >}}
 
-**6. Configure Sensu server**
+Install RabbitMQ:
 
-Run the following to set up a minimal client config:
+{{< highlight shell >}}
+sudo yum install https://dl.bintray.com/rabbitmq/rabbitmq-server-rpm/rabbitmq-server-3.6.12-1.el7.noarch.rpm -y{{< /highlight >}}
+
+Configure RabbitMQ to work with Sensu:
 
 {{< highlight shell >}}
 echo '{
- "transport": {
-   "name": "redis"
- },
- "api": {
-   "host": "127.0.0.1",
-   "port": 4567
- }
-}' | sudo tee /etc/sensu/config.json{{< /highlight >}}
+  "rabbitmq": {
```

Comment:
Sensu's RabbitMQ defaults are user `guest` password `guest` and vhost `/`. The guest user is only allowed to connect from localhost. If we want to advise folks to use non-default credentials and/or vhost, we will also need to provide the relevant rabbitmqctl commands for adding user, setting password, creating vhost and assigning permissions on that vhost to the user.

Reply:
Fixed in https://github.com/sensu/sensu-docs/pull/718/commits/2a3900e41c38f3fc190bc9e2b54ed59d52b630dc